torch/_inductor/triton_heuristics.py

```diff
@@ -460,6 +460,10 @@ def save_cuda_kernel(self, grid, stream, launcher):
             "num_warps": launcher.bin.num_warps,
             "shared_mem": launcher.bin.shared,
             "stream": stream,
+            # User defined triton kernels will have arbitrary kwarg names
+            # This might also add XBLOCK, YBLOCK, ZBLOCK but that is since
+            # no one will read it.
+            **launcher.config.kwargs,
```

Comment:
Can we have run into an edge case that user-defined param names coincide with some of the keys above? If so, should we put `launcher.config.kwargs` into a sub-dict?